poly-commitment/src/tests/commitment.rs

```diff
@@ -81,13 +87,45 @@ impl AggregatedEvaluationProof {
             });
         }
 
+        let combined_inner_product = {
```

Comment:
We should have a function that abstracts away this code because it is duplicated in some places across the PR.

Reply:
Lets do this as a follow-up, there's some urgency around getting this in.